charts/kube-prometheus-stack/templates/prometheus/additionalAlertRelabelConfigs.yaml

```diff
@@ -12,5 +12,5 @@ metadata:
     app: {{ template "kube-prometheus-stack.name" . }}-prometheus-am-relabel-confg
 {{ include "kube-prometheus-stack.labels" . | indent 4 }}
 data:
-  additional-alert-relabel-configs.yaml: {{ toYaml .Values.prometheus.prometheusSpec.additionalAlertRelabelConfigs | b64enc | quote }}
+  additional-alert-relabel-configs.yaml: {{ tpl (toYaml .Values.prometheus.prometheusSpec.additionalAlertRelabelConfigs) . | b64enc | quote }}
```

Comment:
@richardtief could you give me an example/use case here? I would like to know it.

Reply:
We deploy this chart more than once and only activate Prometheus to keep track of community updates.